pkg/model/project.proto

```diff
@@ -97,6 +97,8 @@ message ProjectSSOConfig {
     }
 
     Provider provider = 1 [(validate.rules).enum.defined_only = true];
+    // session ttl for users (munites)
+    int64 session_ttl = 2 [(validate.rules).int64.gt = 0];
```

Comment:
With the rule set like this, I guess project object which is stored in datastore currently without this value will cause error, right?

Reply:
When server take configuration file in, sever does not use getter generated by grpc, so validation does not works.
If the field of sessionTtl does not exist in config file, the zero is set at that time.